pipeline_dp/combiners.py

```diff
@@ -324,10 +328,20 @@ def __init__(self, params: CombinerParams,
         self._metrics_to_compute = metrics_to_compute
 
     def create_accumulator(self, values: Iterable[float]) -> AccumulatorType:
+        min_value = self._params.aggregate_params.min_value
+        max_value = self._params.aggregate_params.max_value
+        squares_min_value, squares_max_value = dp_computations.compute_squares_interval(min_value, max_value)
+
+        middle = dp_computations.compute_middle(min_value, max_value)
+        squares_middle = dp_computations.compute_middle(squares_min_value, squares_max_value)
         clipped_values = np.clip(values,
                                  self._params.aggregate_params.min_value,
                                  self._params.aggregate_params.max_value)
-        return len(values), clipped_values.sum(), (clipped_values**2).sum()
+
+        normalized_values = clipped_values - middle
+        normalized_squared_values = (clipped_values**2) - squares_middle
```

Comment:
Variance of `x_1, ...x_n` does not change when the same number is added to all `x_i`, so variance is not changed with applying normalization. So we can do normalization as `normalized_squared_values = normalized_values**2` and as a result the values will be smaller -> so less rounding, better accuracy. WDYT?

Reply:
Good point. Done.